### PR TITLE
ts7100-usbprod: initial commit

### PR DESCRIPTION
**Pull request by markfeathers**

### CODE REVIEW

**Comment by markfeathers** [comment]
Tested capturing an image, reviewed it by hand.  Also wrote back our latest image and tested booting that.

File: technologic/board/ts7100-usbprod/scripts/blast.sh [hunk 1]
```diff
@@ -0,0 +1,144 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: BSD-2-Clause
+# Copyright (c) 2021-2022 Technologic Systems, Inc. dba embeddedTS
+
+# Edit these variables as needed. When porting, this should be all that
+# needs to change for a new platform. Should be.
+
+# Whole device device node path for eMMC. Assuming it is static each boot.
```
Comment (randytsi369) [inline technologic/board/ts7100-usbprod/scripts/blast.sh:9]: I see "device" here twice is that intentional?

**Review by randytsi369 — APPROVED** [review]
Reviewed, not tested.

File: technologic/board/ts7100-usbprod/scripts/blast.sh (hunk 1 above)
Comment (ts-kris) [reply technologic/board/ts7100-usbprod/scripts/blast.sh:9]: That is a copy/paste error. I bet every other `blast.sh` has that. I'll open an issue about it, thanks.

File: technologic/board/ts7100-usbprod/scripts/genimage.cfg [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		# features are only passed to tune2fs, which doesn't seem to be called when
+		# using genext2fs or mke2fs for some reason.
+		use-mke2fs = "true"
+		extraargs = "-O '^metadata_csum' -O '^64bit' -T default"
+		label = "data"
+	}
+	size = 64M
+}
+
+image ts7250v3-usb-image-replicator.dd {
```
Comment (ts-kris) [inline technologic/board/ts7100-usbprod/scripts/genimage.cfg:19]: This should probably be `ts7100-usb-...`

File: technologic/board/ts7100-usbprod/scripts/post-image.sh [hunk 3]
```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+
+PLAT="ts7250v3"
```
Comment (ts-kris) [inline technologic/board/ts7100-usbprod/scripts/post-image.sh:3]: Should this be `ts7100`?

**Review by ts-kris — CHANGES_REQUESTED** [review]
Can you please update the README as well, and then take a look at the couple of comments I dropped. Not sure if we still want to lump TS-7100 and TS-7250-V3 together which is why I'm not sure if those are correct or not.

Since I've got a 7100 on my desk at the moment, I also want to give this a test before I give final approval/merge.

File: technologic/board/ts7100-usbprod/scripts/blast.sh (hunk 1 above)
Comment (luckshot) [reply technologic/board/ts7100-usbprod/scripts/blast.sh:9]: makes sense to me.

**Review by luckshot — APPROVED** [review]
Kris has some pertinent comments, otherwise looks good.  Didn't test.

**Review by luckshot — APPROVED** [review]

**Review by ts-kris — APPROVED** [review]
IR tested, general TS-7100 config not tested.

**Pull request merged**